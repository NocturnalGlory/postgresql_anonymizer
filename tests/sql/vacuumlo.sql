CREATE EXTENSION IF NOT EXISTS anon CASCADE;

-- INIT

SELECT anon.start_dynamic_masking();


CREATE TABLE t1 (i int);

\! vacuumlo -v contrib_regression

--  CLEAN

DROP EXTENSION anon CASCADE;
DROP SCHEMA mask CASCADE;
DROP TABLE t1;
