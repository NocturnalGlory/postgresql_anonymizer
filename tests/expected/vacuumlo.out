CREATE EXTENSION IF NOT EXISTS anon CASCADE;
-- INIT
SELECT anon.mask_init();
 mask_init 
-----------
 t
(1 row)

CREATE TABLE t1 (i int);
\! vacuumlo -v contrib_regression
Connected to database "contrib_regression"
Successfully removed 0 large objects from database "contrib_regression".
--  CLEAN
DROP EXTENSION anon CASCADE;
NOTICE:  drop cascades to event trigger anon_mask_update
DROP SCHEMA mask CASCADE;
NOTICE:  drop cascades to view mask.t1
DROP TABLE t1;
